### ISSUE TITLE
Language Request: Pine Script

### ISSUE THREAD

**Issue by sigre7944** [PrismJS/prism#3440]
**Language**
Pine Script is TradingView's programming language. It allows traders to create their own trading tools and test them. TradingView is website tool for advanced financial visualisation.

**Additional resources**
Official website: https://www.tradingview.com/
Documentation: https://www.tradingview.com/pine-script-docs/en/v5/Introduction.html#:~:text=Pine%20Script%20is%20TradingView's,that%20you%20can%20then%20backtest.

**Comment by Fred-Vatin** [comment]
Yes, please. As obsidian uses prism, it would allow syntax highlighting for pinescript.
